webui/react/src/hooks/useSettings.ts

```diff
@@ -274,14 +277,14 @@ const useSettings = <T>(config: SettingsConfig, options?: SettingsHookOptions):
           acc.querySettings[key] = undefined;
 
           // If the settings value is invalid, set to undefined.
-          const value = partialSettings[key];
```

Comment:
`partialSettings[key] !== null` should be `partialSettings[key] !== undefined` or `partialSettings[key]`?

Reply:
I'm doing this (`partialSettings[key] !== null`) to avoid a typescript error.